### PR TITLE
Fix ignoring of properties in IssueModelTest

### PR DESCRIPTION
**Pull request by brarrow**
#237 

- [x] Fix ignore prop

### CODE REVIEW

**Comment by greenbarrow** [comment]
On my machine all tests success.
wtf?

**Comment by greenbarrow** [comment]
@malinink I need help :'(

**Comment by brarrow** [comment]
@malinink Konstantin Aleksandrovich, check this PR, please.

File: src/test/java/easytests/core/models/IssueModelTest.java [hunk 1]
```diff
@@ -15,8 +15,7 @@
 
     @Override
     protected ConfigurationBuilder getConfigurationBuilder() {
```
Comment (malinink) [inline src/test/java/easytests/core/models/IssueModelTest.java]: уберите метод, совсем, он тут более не нужен.

File: src/test/java/easytests/support/meanbean/models/SubjectModelFactory.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * @author greenbarrow
+ */
+public class SubjectModelFactory implements Factory<SubjectModelInterface> {
+
+    @Override
+    public SubjectModelInterface create() {
+        return new SubjectModel();
+    }
+}
```
Comment (malinink) [inline src/test/java/easytests/support/meanbean/models/SubjectModelFactory.java]: EOF добавьте

**Review by malinink — CHANGES_REQUESTED** [review]

File: src/test/java/easytests/core/models/IssueModelTest.java (hunk 1 above)
Comment (brarrow) [reply src/test/java/easytests/core/models/IssueModelTest.java]: done

File: src/test/java/easytests/support/meanbean/models/SubjectModelFactory.java (hunk 2 above)
Comment (brarrow) [reply src/test/java/easytests/support/meanbean/models/SubjectModelFactory.java]: done

**Comment by brarrow** [comment]
@malinink Konstantin Aleksandrovich, merge this PR, please.

**Review by malinink — APPROVED** [review]

**Pull request merged**